asurgent/ui
# Markdown - fold blockquotes

# Description

Add option to fold blockquotes in markdown

Fixes https://github.com/asurgent/admin/issues/1197

## Type of change


# How Has This Been Tested?

- Open http://localhost:6006/iframe.html?id=helpers-markdown--main&viewMode=story 
- Notice the "expand" button
- Click it and the block will unfold.
